### ISSUE TITLE
Using SelectableTags in ListView，when sliding listview and tags will redraw, it cause view jitter.

### ISSUE THREAD

**Issue by windows7lake** [Dn-a/flutter_tags#14]
Using SelectableTags in ListView，when sliding listview and tags will redraw, it cause view jitter.
Have any idea about it?

**Comment by windows7lake** [comment]
There is a set of SelectableTags in the listview，after SelectableTags move out screen and then slide into the screen, it happened.

**Comment by Dn-a** [comment]
@windows7lake wrap with container....if it does not work, post you code here

**Comment by milosjovac** [comment]
When wrapped with container there is a render overflow for a couple of ms, so this cannot be used as a solution. 

![image](https://user-images.githubusercontent.com/2808506/57976655-6d50e300-79e5-11e9-876d-7c5ef10be5ac.png)

By debugging I have noticed that `build` method of class `_SelectableTagsState` is always called 2 times. Once with `_width=0.0` and second time with real `_width` value calculated inside `_getwidthContainer()`.

For this reason first tag set is created with every tag in it's own row, and total `height = tagNum * tagHeight`. 

![image](https://user-images.githubusercontent.com/2808506/57976599-6bd2eb00-79e4-11e9-8610-ea3dfec40987.png)

After width is calculated (t < sec) more tags can be in the same row, and height is smaller.

![image](https://user-images.githubusercontent.com/2808506/57976591-34643e80-79e4-11e9-8160-043a9c459369.png)

This transition between `_width=0` and `_width>0` makes ListView/static screens to jitter/jump.

**Solution**: Just return empty container in case width is 0. This is a code snippet from  class `_SelectableTagsState`

```
  @override
  Widget build(BuildContext context) {
    // essential to avoid infinite loop of addPostFrameCallback
    if (MediaQuery.of(context).orientation != _orientation || _width == 0) {
      _getwidthContainer();
      return Container(
        key: _containerKey,
        color: widget.backgroundContainer ?? Colors.white,
      );
    }
```

**Comment by Dn-a** [comment]
@milosjovac Ok thanks.
is a problem due to the fact that to get the exact width it is necessary to first build the layout, then build is called to set the width previously obtained.
I will make the necessary changes.

**Comment by Dn-a** [comment]
@milosjovac @windows7lake fixed in 0.3.2 version.
